### PR TITLE
ref: remove six from tests

### PR DESCRIPTION
**Pull request by joshuarli**
This uses my fork of pyupgrade to remove six from tests.

pyupgrade doesn't rewrite most (other than py3 builtins, which pyupgrade removes. the rest need to be renamed) `six.moves` - those rewriters are in https://github.com/asottile/reorder_python_imports, but that also reorders a lot of our (messy) imports, something that's really nice and I'd like to have, but not the focus right now.

Some references... https://github.com/asottile/pyupgrade/issues/340 https://github.com/asottile/pyupgrade/issues/59

I'll probably just fastmod the rest rather than customize reorder_python_imports or implement in pyupgrade.

#test-getsentry

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
If he's in I'm in. As in, if tests pass then I approve.

File: tests/sentry/api/endpoints/test_organization_activity.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def test_simple(self):
         assert response.status_code == 200, response.content
         assert len(response.data) == 1
         assert response.data[0]["id"] == str(activity.id)
-  
+
```
Comment (joshuarli) [inline tests/sentry/api/endpoints/test_organization_activity.py:25]: Good bot.

Comment (joshuarli) [reply tests/sentry/api/endpoints/test_organization_activity.py:25]: Good @billyvg.

**Comment by billyvg** [comment]
#test-getsentry

**Review by evanpurkhiser — APPROVED** [review]

File: tests/sentry/api/endpoints/test_organization_sentry_app_installations.py [hunk 2]
```diff
@@ -105,7 +103,7 @@ def test_install_published_app(self):
         }
 
         assert response.status_code == 200, response.content
-        assert six.viewitems(expected) <= six.viewitems(response.data)
+        assert expected.items() <= response.data.items()
```
Comment (wedamija) [inline tests/sentry/api/endpoints/test_organization_sentry_app_installations.py:106]: I feel like this test is wrong, lol

Comment (joshuarli) [reply tests/sentry/api/endpoints/test_organization_sentry_app_installations.py:106]: lol

**Pull request merged**